packages/pyright-internal/src/analyzer/service.ts

```diff
@@ -523,7 +523,7 @@ export class AnalyzerService {
             ? optionRoot
             : isString(optionRoot)
             ? Uri.file(optionRoot, this.serviceProvider, /* checkRelative */ true)
-            : Uri.empty();
+            : Uri.file('<default workspace root>', this.serviceProvider);
```

Comment:
Any concerns about `<` and `>` not being legal characters in file paths? At least in Windows file paths.

They are also stream redirection operators for stdin/stdout in some contexts. I can't think of a situation where it would be problematic, but it makes me a little nervous.

How did this work before the URI refactoring?

Reply:
it used to just `''` string. `'<default workspace root>'` is intentionally random text that won't exist but some operation path related operation with `absolute path` will work with correct case sensitivity.

Reply:
Won't this root get combined with other file paths? Whereas the empty would never combine?

Reply:
I updated PR with feedback you gave offline.